### PR TITLE
Data release to version: 2020_09_24_01_02_12

### PR DESCRIPTION
**Pull request by shifucun**
Note there are substantial change in landing page cache data since all the stat var are included there, even there is no data. This is to ensure the mixer can fetch newly added stat var that is not included in Flume

### CODE REVIEW

File: golden_response/staging/get_stat_all/result.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
      ]
     },
     "Count_Person": {
+     "isDcAggregate": true,
```
Comment (pradh) [inline golden_response/staging/get_stat_all/result.json:69]: @tjann should confirm. I'm surprised why this is an aggregate.

**Review by pradh — APPROVED** [review]
Skipped landing-page diffs. Otherwise, LGTM.

File: golden_response/staging/get_stat_all/result.json (hunk 1 above)
Comment (tjann) [reply golden_response/staging/get_stat_all/result.json:69]: yeah this is surprising

Comment (tjann) [reply golden_response/staging/get_stat_all/result.json:69]: Maybe bc this is for all sources, and CensusACS5yrSurvey we aggregate for country IIRC

**Pull request merged**